std/math.d

```diff
@@ -7495,15 +7495,12 @@ if (isFloatingPoint!(F) && isIntegral!(G))
 
     assert(pow(x, neg1) == 1 / x);
 
-    version (X86_64)
-    {
-        pragma(msg, "test disabled on x86_64, see bug 5628");
-    }
-    else version (ARM)
-    {
-        pragma(msg, "test disabled on ARM, see bug 5628");
-    }
-    else
+    // Test disabled on most targets, see bug 5628.
+    version (X86_64)   enum BUG5628 = true;
+    else version (ARM) enum BUG5628 = true;
+    else               enum BUG5628 = false;
```

Comment:
It looks like you have inverted the condition, shouldn't it be:
```D
    version (X86_64)   enum BUG5628 = false;
    else version (ARM) enum BUG5628 = false;
    else               enum BUG5628 = true;
```

Reply:
It's almost as if I'm spending more time on something else in the room instead of looking at my own patches properly. :-)